### ISSUE TITLE
[1.18.2] Lava surface becomes reflective and 'watery' when using with shaders

### ISSUE THREAD

**Issue by RedBeast1959** [Alex-the-666/AlexsMobs#1326]
alexsmobs-1.18.6.jar
citadel-1.11.3-1.18.2.jar
rubidium-0.5.3a.jar
oculus-mc1.18.2-1.2.5a.jar

With current alex's mobs 1.18.2 version, lava with shader looks like reflective 'water', even if I use shader compat config.
This problem doesn't happen in previous 1.18.2 version( alexsmobs-1.18.5.jar )

![2022-09-10_19 02 45](https://user-images.githubusercontent.com/100399264/189478701-3026f681-07a3-45b3-aad2-9659de90efa0.png)

alexsmobs-1.18.5.jar with Complementary Reimagined shader

![2022-09-10_19 06 19](https://user-images.githubusercontent.com/100399264/189478795-d755e23c-17a0-4279-b392-b39381e79a96.png)

alexsmobs-1.18.6.jar with Same shader, same location

**Comment by L0serr** [comment]
Yeah I'm seeing lava rendered as water with SEUS Renewed. Shader Compat option set to true.

**Comment by JianlingRemix** [comment]
> Yeah I'm seeing lava rendered as water with SEUS Renewed. Shader Compat option set to true.

where can i find the Shder Compat?

**Comment by ConnorVex** [comment]
> > Yeah I'm seeing lava rendered as water with SEUS Renewed. Shader Compat option set to true.
> 
> where can i find the Shder Compat?

In your config files, its stored in a .TOML file but even then turning it on doesn't really seem to do much for the issue

**Comment by ducktrucker** [comment]
> > > Yeah I'm seeing lava rendered as water with SEUS Renewed. Shader Compat option set to true.
> > 
> > 
> > where can i find the Shder Compat?
> 
> In your config files, its stored in a .TOML file but even then turning it on doesn't really seem to do much for the issue

i cant find it
